media/breaking-changes/breaking-changes-panel.js

```
@@ -0,0 +1,545 @@
+          }
+
+          // Filter by change type
+          if (filterChangeType !== 'all' && change.changeType !== filterChangeType) {
+            return false;
+          }
+
+          // Filter by search term
+          if (searchTerm) {
+            const searchLower = searchTerm.toLowerCase();
+            return (
+              (change.description && change.description.toLowerCase().includes(searchLower)) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)